0.19.0-release-notes.md

```diff
@@ -21,6 +21,9 @@ and compiled, even if it is not executed:
 -   Temporary assignment on an unset environment variables no longer leave it
     set to an empty string ([#1448](https://b.elv.sh/1448)).
 
+-   Glob expansion that includes broken symlinks no longer short-circuits the
+    expansion ([#1240](https://b.elv.sh/1240)).
```

Comment:
This is a fix but it's also a breaking change - symlinks to directories will now be treated as files, not directories. This breaking change is fine for me - among other things it avoids infinite recursions when expanding `**` symlinks introduce loops in the FS tree. But it should be documented as such.

Reply:
Another good point but the existing behavior is flat out wrong and it's unclear how to document the change in behavior.